fix_failing_tests.py

```
@@ -0,0 +1,153 @@
+        ),
+        (
+            r'assert hasattr\(app, "add_api_event_handler"\)',
+            'assert hasattr(app, "add_event_handler")',
+        ),
+        (
+            r'assert hasattr\(app, "add_api_websocket_event_handler"\)',
+            'assert hasattr(app, "add_event_handler")',
+        ),
+    ]
+
+    original = content
```

Flake8 (7.2.0)

[error] local variable 'original' is assigned to but never used

(F841)